packages/sfui/frameworks/react/components/SfScrollable/SfScrollable.tsx

```diff
@@ -0,0 +1,131 @@
+import { useMemo } from 'react';
+import classNames from 'classnames';
+import {
+  polymorphicForwardRef,
+  SfButton,
+  SfIconChevronLeft,
+  SfIconChevronRight,
+  useScrollable,
+  SfScrollableDirection,
+  SfScrollableButtonsPlacement,
+  type SfScrollableProps,
+} from '@storefront-ui/react';
+
+const defaultScrollableTag = 'div';
+
+const SfScrollable = polymorphicForwardRef<typeof defaultScrollableTag, SfScrollableProps>(
+  (
+    {
+      as,
+      direction = SfScrollableDirection.horizontal,
+      buttonsPlacement = SfScrollableButtonsPlacement.block,
+      activeIndex,
+      reduceMotion,
+      drag,
+      onDragChange,
+      onScroll,
+      onPrev,
+      onNext,
+      className,
+      wrapperClassNames,
+      previousDisabled,
+      nextDisabled,
+      style,
+      children,
+      slotPreviousButton,
+      slotNextButton,
+      ...attributes
+    },
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    ref,
+  ) => {
+    const Tag = as || defaultScrollableTag;
+
+    const sliderOptions = useMemo(
+      () => ({
+        direction,
+        activeIndex,
+        reduceMotion,
+        drag,
+        onDragChange,
+        onScroll,
+        onPrev,
+        onNext,
+      }),
+      [direction, activeIndex, reduceMotion, drag, onDragChange, onScroll, onPrev, onNext],
+    );
+
+    const { getContainerProps, state, getNextButtonProps, getPrevButtonProps } = useScrollable(sliderOptions);
+
+    const changeDisabledClass = (isDisabled: boolean) =>
+      isDisabled ? '!ring-gray-200 !text-disabled-500' : '!ring-neutral-500 !text-neutral-500';
```

Comment:
Can we use `ring-neutral-200` here? 
```suggestion
      isDisabled ? '!ring-neutral-200 !text-disabled-500' : '!ring-neutral-500 !text-neutral-500';
```

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2571/commits/1ae9490399f2c745ca6b135135141730d3eec6f9 i used `disabled` color as in figma, thanks for pointing it out